nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/service/canvas-utils.service.ts

```diff
@@ -268,7 +270,7 @@ export class CanvasUtils {
     public canModify(selection: any): boolean {
         const selectionSize = selection.size();
         const writableSize = selection
-            .filter(function (d: any) {
+            .filter(function(d: any) {
```

Comment:
There are several formatting changes in this file. Was this intentional?

Reply:
must just be from running prettier. I didn't manually do these.